zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py

```diff
@@ -76,7 +80,8 @@ def execute(self, stmt):
 
             return result
 
-        finally:
-            cl.shutdown()
+        except Exception:
```

Comment:
No need to catch exceptions here, let it bubble up and let the user decide how to handle them